### PR TITLE
Fix to add cpu architecture suffix to artifact name of cljstyle

### PR DESCRIPTION
**Pull request by hatappo**
To fix the issue: https://github.com/DeLaGuardo/setup-clojure/issues/100

Thank you for maintaining a great workflow like this repository. We always appreciate your work.

### CODE REVIEW

**Comment by DeLaGuardo** [comment]
LGTM! thanks)

Before merging, could you run `npm run all` and commit changed sources? This command will run a few tests and compile .ts to .js files.

**Comment by hatappo** [comment]
I did not take care of the test cases. My apologies.
(Should I add a test case for boundary conditions by version number? If so, please let me know.)

I modified the test code as well to accommodate the logic changes and also committed the built artifacts.

File: __tests__/cljstyle.test.ts [hunk 1]
```diff
@@ -77,9 +77,9 @@ describe('cljstyle tests', () => {
   describe('getArtifactUrl', () => {
     test.each`
       platform    | artifact
-      ${'darwin'} | ${`cljstyle_1.2.3_macos.zip`}
-      ${'linux'}  | ${`cljstyle_1.2.3_linux.zip`}
-      ${'foobar'} | ${`cljstyle_1.2.3_linux.zip`}
+      ${'darwin'} | ${`cljstyle_1.2.3_macos_amd64.zip`}
+      ${'linux'}  | ${`cljstyle_1.2.3_linux_amd64.zip`}
+      ${'foobar'} | ${`cljstyle_1.2.3_linux_amd64.zip`}
```
[the remark this answers is not in the document]
Comment (hatappo) [reply __tests__/cljstyle.test.ts]: I added a parameter to test older versions as well. Does this change deviate from your intended implementation?

[the remark this answers is not in the document]
Comment (DeLaGuardo) [reply __tests__/cljstyle.test.ts]: no, it looks perfect, thanks

**Review by DeLaGuardo — APPROVED** [review]
